### PR TITLE
HW03 - Add 3ArraysIndexIndex2Index3Average app

### PR DESCRIPTION
**Pull request by Serge-prospect**
Please review HW03 - 3ArraysIndexIndex2Index3Average app

### CODE REVIEW

File: 3ArraysIndexIndex2Index3Average/3ArraysIndexIndex2Index3Average/Program.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+namespace _3ArraysIndexIndex2Index3Average
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            //Array fulfilled by indexes
+            int[] array1Degree = new int[10];
+            float sumArray1Degree = 0f;            
+            for (int index = 0; index < array1Degree.Length; index++)
```
Comment (EgorAntonovich) [inline 3ArraysIndexIndex2Index3Average/3ArraysIndexIndex2Index3Average/Program.cs]: Please, simplify 3 cycles into 1 function.

File: 3ArraysIndexIndex2Index3Average/3ArraysIndexIndex2Index3Average/Program.cs [hunk 2]
```diff
@@ -0,0 +1,46 @@
+﻿using System;
+
+namespace _3ArraysIndexIndex2Index3Average
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            //Array fulfilled by indexes
+            int[] array1Degree = new int[10];
```
Comment (EgorAntonovich) [inline 3ArraysIndexIndex2Index3Average/3ArraysIndexIndex2Index3Average/Program.cs]: Plese, create function for init arrays

File: 3ArraysIndexIndex2Index3Average/3ArraysIndexIndex2Index3Average/Program.cs [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+            float sumArray1Degree = 0f;            
+            for (int index = 0; index < array1Degree.Length; index++)
+            {
+                array1Degree[index] = (int)Math.Pow(index, 1);
+                sumArray1Degree += array1Degree[index];
+            }
+            float averageArray1Degree = sumArray1Degree / array1Degree.Length;
+            Console.WriteLine("Array fulfilled by indexes: [{0}]", String.Join(", ", array1Degree));
```
Comment (EgorAntonovich) [inline 3ArraysIndexIndex2Index3Average/3ArraysIndexIndex2Index3Average/Program.cs]: Plese, create function for output info about arrays

**Review by EgorAntonovich — CHANGES_REQUESTED** [review]
You need full refactoring

File: 3ArraysIndexIndex2Index3Average/3ArraysIndexIndex2Index3Average/Program.cs (hunk 3 above)
Comment (Serge-prospect) [reply 3ArraysIndexIndex2Index3Average/3ArraysIndexIndex2Index3Average/Program.cs]: Refactored by using methods.

**Review by EgorAntonovich — APPROVED** [review]
Looks good for me

**Pull request merged**